ext/AvizExt/viz/viz.jl

```diff
@@ -81,16 +81,10 @@ function _calc_gridsize(n_factors::Int64; max_cols::Int64=4)::Tuple{Int64,Int64}
     return n_rows, n_cols
 end
 
-function confint(
-    data::NamedDimsArray, dimension::Symbol
-)::Tuple{Vector{Float64},Vector{Float64},Vector{Float64}}
+function confint(data::NamedDimsArray, dimension::Symbol)::Vector{Vector{Float64}}
     data_slices = Slices(data, NamedDims.dim(data, dimension))
-
-    y_lower = quantile.(data_slices, [0.025])
-    y_upper = quantile.(data_slices, [0.975])
-    y_median = median.(data_slices)
-
-    return y_lower, y_upper, y_median
+    quantiles::Matrix{Float64} = quantile.(data_slices, [[0.025] [0.5] [0.975]])
```

Comment:
Just wondering/checking - is the vector of vectors needed when working with slices?

```julia
quantile.(data_slices, [0.025, 0.5, 0.975])
```

Reply:
This `[[0.025] [0.5] [0.975]]` returns a Matrix (in Julia a Matrix is not the same as a Vector of Vectors, right?). And no, just passing the vector directly doesn't work:

```julia
julia> quantile.(data_slices, [0.025, 0.5, 0.975])
ERROR: DimensionMismatch: arrays could not be broadcast to a common size; got a dimension with lengths 75 and 3
```

And just in case you're wondering, a real vector of vectors work but is not as easy to work with then the matrix:

```julia
julia> quantile.(data_slices, [[0.025, 0.5, 0.975]])
75-element Vector{Vector{Float64}}:
 [6.5823215e6, 6.5823215e6, 6.5823215e6]
 [2.790339125e6, 5.5685215e6, 6.1646672e6]
 [2.332284275e6, 4.3975435e6, 5.8010557e6]
 [2.313613325e6, 3.744976e6, 5.34081515e6]
 ⋮
 [251626.7109375, 357739.53125, 534253.89375]
 [236002.25468749998, 363243.15625, 550407.7937500001]
 [208631.68437499998, 353628.40625, 527181.8625]
```

Reply:
> This [[0.025] [0.5] [0.975]] returns a Matrix (in Julia a Matrix is not the same as a Vector of Vectors, right?).

Ah, you don't need individual single element vectors then, just a space will do.
```julia
julia> [[0.025] [0.5] [0.975]]
1×3 Matrix{Float64}:
 0.025  0.5  0.975

julia> [0.025 0.5 0.975]
1×3 Matrix{Float64}:
 0.025  0.5  0.975
```